cmd/submit.go

```diff
@@ -207,8 +231,8 @@ figuring things out if necessary.
 		}
 
 		if solution.AutoApprove == true {
-			fmt.Fprintf(Out, "Your solution has been submitted " +
-				"successfully and has been auto-approved. You can complete " +
+			fmt.Fprintf(Out, "Your solution has been submitted "+
+				"successfully and has been auto-approved. You can complete "+
 				"the exercise and unlock the next core exercise at %s\n",
```

Comment:
Were these changes introduced by a change to `gofmt`? If not, can we revert them, since they're not related to this change?

Reply:
It was, as part of [this](https://github.com/exercism/cli/pull/503/commits/6e09ba18d1536f432563d0ff44ff951cf7d2cab5) commit. Not sure why it didn't change before.

Reply:
```
>>> go version
go version go1.9.4 linux/amd64
```